webcommon/typescript.editor/external/typescript-5.5.4-license.txt

```diff
@@ -1,6 +1,6 @@
 Name: typescript
 Description: TypeScript is a language for application scale JavaScript development
-Version: 4.1.4
+Version: 5.5.4
```

Comment:
Sorry, but this seems highly suspicious to me. The `typescript-5.5.4.zip` file contains `typescript/ThirdPartyNoticeText.txt`, and I don't see the content of the file here. Is there a reason to believe the `typescript/ThirdPartyNoticeText.txt` does not apply here?

Reply:
Valid point. I admit, I did not check, on the other hand, the relevant file would be NOTICE, but so be it.